app/src/androidTest/java/org/mozilla/fenix/ui/util/Strings.kt

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ui.util
+
+const val STRING_ONBOARDING_ACCOUNT_SIGN_IN_HEADER = "Sync Firefox between devices"
+const val STRING_ONBOARDING_TRACKING_PROTECTION_HEADER = "Always-on privacy"
+const val STRING_ONBOARDING_TOOLBAR_PLACEMENT_HEADER = "Pick your toolbar placement"
```

Comment:
Why do we need these for the UI tests? Why can't we use strings.xml?

Reply:
We want to re-use text as is heavily used in UI tests making pretty difficult when updates like this one happen, for more context see https://github.com/mozilla-mobile/fenix/pull/18550#discussion_r598872697. There are some functions that require strings instead of string id like `scrollToElementByText`.